frontend/src/components/explorer/mapViewer/DataOverlayValidation.vue

```diff
@@ -0,0 +1,70 @@
+<template>
+  <div class="file is-centered mb-2">
+    <label class="file-label">
+      <input id="anId" class="file-input" type="file" name="resume" @change="validateFile"
```

Comment:
I'm not sure if this id is needed?

```suggestion
      <input class="file-input" type="file" name="resume" @change="validateFile"
```

Reply:
Good catch. I'm thinking it makes sense to remove name="resume" as well?

Reply:
Yes, it looks like there is no other reference to `resume` in the codebase.